src/actions/heap/heap.ts

```diff
@@ -58,19 +70,13 @@ export class HeapAction extends Hub.Action {
   supportedActionTypes = [Hub.ActionType.Query]
   supportedPropertyTypes = [HeapPropertyTypes.User, HeapPropertyTypes.Account]
   usesStreaming = true
-  supportedFormats = (request: Hub.ActionRequest) => {
```

Comment:
why was this removed?

Reply:
I don't think it's used. I checked other actions. 

https://github.com/looker/actions/blob/e2d0d8c0261bc06d48b2a98e4327a80dcbc5ef50/src/actions/google/drive/google_drive.ts#L16-L21

https://github.com/heap/actions/blob/master/src/actions/slack/slack.ts#L23-L33

Should I add a `minimumSupportedLookerVersion`? Frankly, I don't find a dev doc.

Reply:
https://docs.looker.com/sharing-and-publishing/action-hub#supported_data_formats

Reply:
sounds good, I put it back.